### PR TITLE
Connect client server

### PR DESCRIPTION
**Pull request by karmelyoei**

### CODE REVIEW

File: server/controllers/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+
+const router = express.Router();
+console.log('index router from controller');
```
Comment (RandInaim) [inline server/controllers/index.js]: please remove console log

File: server/controllers/index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+router.get('/', (request, response) => {
```
Comment (RandInaim) [inline server/controllers/index.js]: please change the route and also in axios request

File: client/src/App.js [hunk 3]
```diff
@@ -1,26 +1,28 @@
...
-import logo from './logo.svg';
+import React, { Fragment } from 'react';
+import {
+	BrowserRouter as Router,
+	Route,
+	Switch,
+	Redirect,
+} from 'react-router-dom';
+
+import Home from './Screens/Home';
```
Comment (RandInaim) [inline client/src/App.js:9]: please rename Screens folder to screens

**Comment by karmelyoei** [comment]
Done but please don't close the issue let me put some notes for my team

**Review by RandInaim — APPROVED** [review]

**Pull request merged**